Makefile

```diff
@@ -528,6 +538,11 @@ lr/docs/markdown: providers/lr
 		--description "The Operating Systems (OS) resource pack lets you use MQL to query and assess the security of your operating system packages and configuration." \
 		--docs-file providers/os/resources/os.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/os-pack
+	./lr markdown providers/shodan/resources/shodan.lr \
+		--pack-name "Shodan" \
+		--description "The Shodan resource pack lets you use MQL to query and assess the security of your Shodan service." \
```

Comment:
We probably want to update this to describe what data you're getting from Shodan in these resources